README.md

```diff
@@ -4,29 +4,72 @@
 [![crates.io](https://meritbadge.herokuapp.com/wrangler)](https://crates.io/crates/wrangler)
 [![Build Status](https://dev.azure.com/ashleygwilliams/wrangler/_apis/build/status/cloudflare.wrangler?branchName=master)](https://dev.azure.com/ashleygwilliams/wrangler/_build/latest?definitionId=1&branchName=master)
 
-✨ CHECK OUT THE [TUTORIAL](https://developers.cloudflare.com/workers/webassembly/tutorial/) ✨
-
 ## 🎙️ Commands
 
-`wrangler` is a CLI tool designed for folks who are interested in using Rust-generated WebAssembly on
-Cloudflare Workers. This tool gives you the following commands:
+`wrangler` is a CLI tool designed for folks who are interested in using Cloudflare workers.
```

Comment:
Could we move this into the top-level (above "Commands") and make it slightly more declarative about what `wrangler` is? 

> `wrangler` is a CLI tool for developing and publishing applications with Cloudflare Workers.

Open to copy there, just generally think we could use a one-liner about _what_ wrangler is at the beginning of the README